code/controllers/configuration/entries/general.dm

```diff
@@ -597,6 +597,8 @@
 	default = -1
 	min_val = 0
 
+/datum/config_entry/flag/setup_bypass_player_check
+
```

Comment:
this should be reversed in the code, a config to enable the check, defaulting to false, and enabled in the default config.txt, im gonna want this disabled on tg because i want people to be able to start the round with only observer or people in the lobby if thats what the players want to do.

Reply:
Okay